programs/server/Server.cpp

```diff
@@ -2038,6 +2039,7 @@ try
 
             global_context->setMaxTableSizeToDrop(new_server_settings[ServerSetting::max_table_size_to_drop]);
             global_context->setMaxPartitionSizeToDrop(new_server_settings[ServerSetting::max_partition_size_to_drop]);
+            global_context->setMaxNamedCollectionNumToWarn(new_server_settings[ServerSetting::max_named_collection_num_to_warn]);
```

Comment:
Here and in other places.
```suggestion
            global_context->setMaxNamedCollectionsToWarn(new_server_settings[ServerSetting::max_named_collection_num_to_warn]);
```